tests/integration_test/services_test.py

```diff
@@ -5573,9 +5564,21 @@ def _password_reset(*args, **kwargs):
 
 
 def test_login_limit_exceeded():
+    response_one = client.post(
+        "/api/auth/login",
+        data={"username": "integration@demo.ai", "password": "Welcome@3010"},
+    ).json()
+    response_two = client.post(
+        "/api/auth/login",
+        data={"username": "integration@demo.ai", "password": "Welcome@3010"},
+    ).json()
+    response_three = client.post(
+        "/api/auth/login",
+        data={"username": "integration@demo.ai", "password": "Welcome@3010"},
+    ).json()
     response = client.post(
         "/api/auth/login",
-        data={"username": "integration@demo.ai", "password": "Welcome@10"},
+        data={"username": "integration@demo.ai", "password": "Welcome@3010"},
     ).json()
     print(response)
     assert not response['success']
```

Comment:
change message to "Account frozen due to too many unsuccessful login attempts. Please come back in ..... "